stable_baselines3/common/distributions.py

```diff
@@ -658,7 +658,6 @@ def make_proba_distribution(
         dist_kwargs = {}
 
     if isinstance(action_space, spaces.Box):
-        assert len(action_space.shape) == 1, "Error: the action space must be a vector"
```

Comment:
interesting, it still works when the assert is False?

Reply:
yes, that is the point of the PR:

The distribution still works in 1D, thank to `get_action_dim`:

https://github.com/DLR-RM/stable-baselines3/blob/f6fefc3d139984c5544086ff728ca4ebfce4c0c5/stable_baselines3/common/distributions.py#L662

Then, when predicting, the distribution returns a 1D array that we need to reshape to the action shape in `BasePolicy` :

https://github.com/DLR-RM/stable-baselines3/blob/f6fefc3d139984c5544086ff728ca4ebfce4c0c5/stable_baselines3/common/policies.py#L340

and also in `ActorCriticPolicy`:

https://github.com/DLR-RM/stable-baselines3/blob/f6fefc3d139984c5544086ff728ca4ebfce4c0c5/stable_baselines3/common/policies.py#L595